docs/UsersGuide/source/Quickstart.rst

```diff
@@ -1,320 +1,325 @@
-You may need to modify certain variables such as the path to NCEP libraries for your individual platform,
-or use ``setenv`` rather than ``export`` depending on your environment:
+For those working on non-cloud-based systems, skip to :numref:`Step %s <WorkOnHPC>`. Users building the SRW App using NOAA's Cloud resources must complete a few additional steps to ensure that the SRW App builds and runs correctly. 
 
-.. code-block:: console
+On NOAA Cloud systems, certain environment variables must be set *before* building the container:
+   
+.. code-block:: 
 
-   $ ls -l env/
-      -rw-rw-r-- 1 user ral 1062 Apr  27 10:09 build_cheyenne_gnu.env
-      -rw-rw-r-- 1 user ral 1061 Apr  27 10:09 build_cheyenne_intel.env
-      -rw-rw-r-- 1 user ral 1023 Apr  27 10:09 build_hera_intel.env
-      -rw-rw-r-- 1 user ral 1017 Apr  27 10:09 build_jet_intel.env
+   sudo su
+   export SINGULARITY_CACHEDIR=/lustre/cache
+   export SINGULARITY_TEMPDIR=/lustre/tmp
 
-Build the Executables
-=====================
-Build the executables as follows:
+If the ``cache`` and ``tmp`` directories do not exist already, they must be created with a ``mkdir`` command. 
 
-.. code-block:: console
+.. note:: 
+   ``/lustre`` is a fast but non-persistent file system used on NOAA cloud systems. To retain work completed in this directory, `tar the files <https://www.howtogeek.com/248780/how-to-compress-and-extract-files-using-the-tar-command-on-linux/>`__ and move them to the ``/contrib`` directory, which is much slower but persistent.
 
-   mkdir build
-   cd build
+.. _WorkOnHPC:
+
+Working on HPC Systems
+--------------------------
 
-Run ``cmake`` to set up the ``Makefile``, then run ``make``:
+Those *not* working on HPC systems may skip to the :ref:`next step <BuildC>`. 
+On HPC systems (including NOAA's Cloud platforms), allocate a compute node on which to run the SRW App. On NOAA's Cloud platforms, the following commands will allocate a compute node:
 
 .. code-block:: console
 
-   cmake .. -DCMAKE_INSTALL_PREFIX=..
-   make -j 4  >& build.out &
+   salloc -N 1 
+   module load gnu openmpi
+   mpirun -n 1 hostname
+   ssh <hostname>
 
-Output from the build will be in the ``ufs-srweather-app/build/build.out`` file.
-When the build completes, you should see the forecast model executable ``NEMS.exe`` and eleven
-pre- and post-processing executables in the ``ufs-srweather-app/bin`` directory which are
-described in :numref:`Table %s <ExecDescription>`.
+The third command will output a hostname. Replace ``<hostname>`` in the last command with the output from the third command. After "ssh-ing" to the compute node in the last command, build and run the SRW App from that node. 
 
-Generate the Workflow Experiment
-================================
-Generating the workflow experiment requires three steps:
+The appropriate commands on other Level 1 platforms will vary, and users should consult the documentation for those platforms. 
```

Comment:
The RDHPCS docs are not publicly available. Should I link anyway? I could refer them to the RDHPCS help email, too/instead... that might actually be more useful, since I'm not sure that the RDHPCS docs list actual code commands to run to allocate a compute node. This was a conversation we had a little while ago. Alternatively, I could contact the help desk or @mark-a-potts and add the commands for each Level 1 system. Thoughts?

Reply:
If it gets too complicated, it is fine as is.

Reply:
Added a link to [this page](https://github.com/ufs-community/ufs-srweather-app/wiki/Supported-Platforms-and-Compilers), which has links to the docs for various platforms.